dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnector.java

```diff
@@ -75,6 +79,46 @@ public abstract class AbstractOracleConnector extends AbstractJdbcConnector {
 
   public static final int OPT_PORT_DEFAULT = 1521;
 
+  protected enum CommonOracleConnectorProperty implements ConnectorProperty {
+    USE_FETCH_SIZE_WITH_LONG_COLUMN(
+        "oracle.useFetchSizeWithLongColumn",
```

Comment:
We do not have any flags that use camelCase. Existing examples:

```
-Dteradata-logs.query-logs-table
-Dteradata.metadata.tablesizev.max-rows
-Dsnowflake.metadata.databases.where
-Dteradata.tmode
-Dsnowflake.query_acceleration_history.query
```

There is no naming convention, but I would suggest following one of the existing case formats.